addons/l10n_in_sale/wizard/sale_make_invoice_advance.py

```diff
@@ -7,8 +7,8 @@
 class SaleAdvancePaymentInv(models.TransientModel):
     _inherit = "sale.advance.payment.inv"
 
-    def _create_invoice(self, order, so_line, amount):
-        invoice = super()._create_invoice(order, so_line, amount)
```

Comment:
well you've taken my 'pseudo-code' and ran with it, but I would expect you to test it before :p
=> you should call the method on `super`: `res = super()._create_invoice(order, so_line, amount)`; otherwise the call to super is an object...
=> the dict assignation require the journal's id, not the record